backend/src/Controller/AutocompleteController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AutocompleteController extends AbstractController
+{
+    /**
+     * @Route("/ac/{place}", name="autocomplete", methods={"GET"})
+     */
+    public function index(string $place): JsonResponse
+    {
+        $google_url = "https://maps.googleapis.com/maps/api/place/autocomplete/json?input=$place&key=AIzaSyDZJTQ_zk-aXNr5gH8Si82_AOHiUmVXDJg&language=de";
+
+        $json =  file_get_contents($google_url);
+        $responseBody = json_decode($json);
+
+        header('Content-Type: application/json');
+        header('Access-Control-Allow-Origin: *');
```

Comment:
There is no need to allow all origins. Actual it is a security risk to do that 😮.
I expect that you will have a single origin in your deployment like `https://alex-project.com` which will serve your frontend (React App) and your API (PHP). In this case, CORS headers are not required and should only same-origin request. If you have separate deployments like `https://api.alex-project.com` (API)  and `https://alex-project.com` (Frontend), the CORS headers should be set to `alex-project.com`. 

In development, you will face a similar scenario, because you have two host (actual two different ports on localhost). 
The good thing is, that Create-React-App has a [build-in Proxy](https://create-react-app.dev/docs/proxying-api-requests-in-development/) to allow CORS requests.
You only have to add `"proxy": "http://localhost:4000",` (replace with your local PHP server) to your frontend/package.json and restart it.
 
I know that this is a complex topic, but it's worth to understand the meaning of CORS headers to get a better understanding of web security 😎

Reply:
This topic does not seem to be an easy one either (didn't work right away.) I will take a closer look at it. Thank you!